scripts/compare-llama-bench.py

```diff
@@ -241,16 +238,12 @@ def get_commit_hexsha8(name):
             break
 
     if hexsha8_compare is None:
-        print("ERROR: No compare target was provided and did not find data for any non-master commits.")
-        print()
+        logger.error("No compare target was provided and did not find data for any non-master commits.")
         parser.print_help()
         sys.exit(1)
 else:
-    print(
-        "ERROR: No compare target was provided and the current working directory "
-        "is not part of a git repository from which a compare target could be inferred."
-    )
-    print()
+    logger.error("No compare target was provided and the current working directory "
+                 "is not part of a git repository from which a compare target could be inferred.")
     parser.print_help()
```

Comment:
For readability, I think we should keep the blank line (`print()`) before calls to print_help. print_usage is short so it shouldn't matter.
```suggestion
    logger.error("No compare target was provided and the current working directory "
                 "is not part of a git repository from which a compare target could be inferred.\n")
    parser.print_help()
```